sw/supervision/python/.gitignore

```diff
@@ -0,0 +1 @@
+generated/
```

Comment:
maybe it is easier to include the correct generated files (but still keep the .ui files for developers) so standard user can use this without installing development libraries and running the makefile

Reply:
ok, done.